pkgs/by-name/hm/hmrc-paye-tools/package.nix

```
@@ -0,0 +1,84 @@
+{ lib
+, stdenv
+, fetchurl
+, buildFHSEnv
+, unzip
+}:
+
+let
```

Consider introducing a dedicated `version` variable in the let-in
```suggestion
let
  version = "24.1.24086.542";
```